machine/wnc_x86_64/i2ctools/i2ctools-wnc.patch

```diff
@@ -0,0 +1,287 @@
+diff --git a/sys_eeprom/onie_tlvinfo.c b/sys_eeprom/onie_tlvinfo.c
```

Comment:
We are no longer accepting patches against i2ctools as that approach is deprecated.

All EEPROM access for x86 machines will use BusyBox as described here:
http://lists.opencompute.org/pipermail/opencompute-onie/2015-March/000512.html